deploy/helm/values.yaml

```diff
@@ -14,6 +14,11 @@ targetNamespaces: ""
 # mode, i.e. when the targetNamespaces values is a blank string.
 excludeNamespaces: "kube-system,{{ .Release.Namespace }}"
 
+# targetWorkloads is a comma seperated list of Kubernetes workload resources
+# to be included in the vulnerability and config-audit scans
+# if left blank, all workload resources will be scanned
+targetWorkloads: "Pod,ReplicaSet,ReplicationController,StatefulSet,DaemonSet,CronJob,Job"
```

Comment:
Term thing, make it lower case if we refer it as `resources` (in comments) or keep it like this 
and refer it as `Kinds` 
```suggestion
targetWorkloads: "pod,replicaset,replicationcontroller,statefulset,daemonset,cronjob,job"
```

Reply:
Changed these to lower-case in the helm values and updated the static manifest to reflect it.